packages/io/amf-deserializer/index.js

```diff
@@ -19,6 +19,7 @@ History:
 //
 // //////////////////////////////////////////
 
+const version = require('./package.json').version
```

Comment:
hmm this is nice & simple but it changes the significance of the 'version'  compared to previous versions (ie jscad version versus deserializer version), I think it is ok though :)

Reply:
we should just make that clear

Reply:
good idea. i changed the metadata information about the producer, specifically saying JSCAD IO library. i'll update the other IO libraries as well.